pkg/domain/iam/entities/group.go

```diff
@@ -18,12 +18,23 @@ type Group struct {
 	ID            uuid.UUID            `json:"id" bson:"_id"`
```

Comment:
It would be interesting to have a readable field to identify the group in parallel with the id so that in the future, access control, if a group is created, does not require the need to place a uuid in the middle of the code.